src/cgame/cg_weapons.cpp

```diff
@@ -941,6 +959,11 @@ static bool CG_ParseWeaponFile( const char *filename, int weapon, weaponInfo_t *
 		return false;
 	}
 
+	if ( wi->crossHairSizeNoBorder == 0.0f && wi->crossHair ) {
```

Comment:
Can we get rid of the warning please? It's going to spam everyone until the next release is out.

Reply:
So we shall release soon

Reply:
This shouldn't really affect anyone though? To get the warning, one would need to run master cgame without master res-weapons. If a server operator uses this change, they're expected to use the updated res-weapons as well. (I assume that no one other than developers really runs master gamelogic on a local server with release assets)

I can comment out the warning until the next release though.

Reply:
> To get the warning, one would need to run master cgame without master res-weapons.

Exactly. Running with every asset package from source would be a pain in the ass so we want it to be usable with just the master Unvanquished repo.

Reply:
I don't think it's that big of a deal to require the latest resources. Annoying for sure, but I think we're probably going to be ready for a release soon enough.

Reply:
It is a big deal to set up your environment with the latest asset packages because many of them require compilation with the asset toolchain and can't be used directly from source. Assuming that a release will happen very soon is a bad idea.

Reply:
I guess "require" was a strong word. A warning log is hardly a big deal.

Reply:
That would be 15 or so warning logs, one for each weapon.

This warning doesn't really seem useful for preventing mistakes anyway... The number would need to be updated whenever the image is changed, but there's no way to detect that.

Reply:
>That would be 15 or so warning logs, one for each weapon.

currently we have a lot of warnings on the games
and there is this too : "[further messages like this will be suppressed] 
"
Then in fact, the result is something like this : 

![Screenshot_20240217_082851](https://github.com/Unvanquished/Unvanquished/assets/148749571/374a8600-2a11-4b13-b8d5-8632d699f697)

It is not a real problem. we have warnings too, and we just ignore them.

![Screenshot_20240217_083131](https://github.com/Unvanquished/Unvanquished/assets/148749571/52ea1e14-6388-4ec2-87b4-e06aeeb199fc)

like this warnings messages. I think that one year ago we had them too. 
Then warnings is not a problem.

Reply:
All right, full spam ahead if that's what you guys want :stuck_out_tongue: